resources/custom-index-name/custom-index-name.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+/**
+ * Read the documentation (https://strapi.io/documentation/developer-docs/latest/concepts/models.html#lifecycle-hooks)
```

Comment:
404 for this link

Reply:
The link is present everywhere! Making a PR to change them all